chain/epoch_manager/README.md

```diff
@@ -0,0 +1,40 @@
+# near-epoch-manager crate
+
+Epoch manager crate is responsible for code related to epochs and epoch switching.
+An epoch is a unit of time when validators of the network remain constant.
```

Comment:
```suggestion
An epoch is a unit of time when the set of validators of the network remains constant.
```

Reply:
Actually, how do epoch interract with sharding? Am I correct that the epoch is the same for all shards, but each shard has its own set of validators?

Reply:
theoretically yes, but currently all shards have same set of validators

Reply:
yes, epoch is the same for all shards. Number of shards can only change during epoch change.